benchmark/CMakeLists.txt

```diff
@@ -50,15 +50,23 @@ macro(add_dataset datasets dataset path_to_template range)
 endmacro()
 
 add_custom_target(benchmarks)
-macro(add_chart target)
+macro(add_chart type case)
     cmake_parse_arguments(ARGS "" "" "DATASETS" ${ARGN})
 
+    set(type_target "benchmarks.${type}")
```

Comment:
Ah, actually we probably want `benchmarks.hetero` (because it's many), but `benchmark.hetero.fold_left` (because it's only one). Or is it more consistent to be plural/singular all the time?